pkg/executor/test/fktest/foreign_key_test.go

```diff
@@ -1680,7 +1680,7 @@ func TestForeignKeyOnUpdateCascade(t *testing.T) {
 			tk.MustExec("update t1 set a=101 where id = 1")
 			tk.MustExec("insert into t2 (id, a, b, name) values (1, 1, 1, 'a')")
 			tk.MustExec("update t1 set b=102 where id = 2")
-			tk.MustQuery("select * from t1").Check(testkit.Rows("1 101 1", "2 1 102"))
+			tk.MustQuery("select * from t1").Check(testkit.Rows("2 1 102", "1 101 1"))
```

Comment:
To make the test result stable, without require ORDER BY, just add `.Sort()`
```suggestion
			tk.MustQuery("select * from t1").Sort().Check(testkit.Rows("1 101 1", "2 1 102"))
```